.github/workflows/rav1e.yml

```diff
@@ -76,6 +77,8 @@ jobs:
            toolchain: stable
          - conf: check-extra-feats
            toolchain: stable
+         - conf: fuzz
+           toolchain: nightly-2019-12-02
```

Comment:
I'll try to get upstream to put back in stable tier the other checks.

Reply:
Note that this is the same nightly we have pinned for `grcov`.

Reply:
It is good for the time being for sure. Now upstream knows that there is a problem